.azuredevops/deploy-platform-core.yaml

```diff
@@ -110,3 +110,6 @@ extends:
             privateEndpointDnsRecordsForResources:
               - resourceName: $(infraResourceNamePrefix)$(nc_resource_containerregistry)$(nc_instance_regionid)01
                 resourceGroupName: $(servicesResourceGroup)
+          - name: flexible-server
+            path: infra/postgre-sql
+            resourceGroupName: $(servicesResourceGroup)
```

Comment:
should this be within the 'dbs' resource group for services?

Reply:
yes, I will change it to dbs

Reply:
Done